heron/spi/src/java/com/twitter/heron/spi/packing/InstanceId.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.twitter.heron.spi.packing;
+
+/**
+ * Encapsulates the unique components of an Instance identifier.
+ */
+public final class InstanceId {
```

Comment:
the instance Id used to container container index also `String.format("%d:%s:%d:%d", containerIdx, componentName, instanceIdx, componentIdx)`. Is it not needed any more?